tunnel/config.go

```diff
@@ -50,13 +50,19 @@ func (r SSHConfigFile) Get(host string) *SSHHost {
 		user = ""
 	}
 
+	local := r.getLocal(host)
```

Comment:
How about having one method to get the configuration: 

`func (r SSHConfigFile) getLocalForward() (*LocalFoward, error) {`

It would simplify the process of finding errors while grabbing the configuration, including misconfiguration like having only the local but not remote.

Reply:
Yes, you're right, I'll change it.